Block/Adminhtml/Support/Edit/Tab/ConfigurationSettings.php

```diff
@@ -110,7 +110,7 @@ protected function _prepareForm()
             'pspReference',
             'text',
             [
-                'name' => 'pspReference',
+                'name' => 'pspReferenceConfigurationSettings',
```

Comment:
Let's make each tab an array of fields: 
```suggestion
                'name' => 'configuration[pspReference]',
```